### DISCUSSION TITLE
how to print dice of multi class segmentation like nnunet?

### DISCUSSION THREAD

**Discussion by jjhhan** [Project-MONAI/MONAI#4231] [Q&A] [answered]
thanks for brilliant work！

![image](https://user-images.githubusercontent.com/41336761/166889361-3aaf9eb2-285d-48bf-b63c-0bc2a6986db2.png)  code currently
want to print like this
![image](https://user-images.githubusercontent.com/41336761/166890777-a48bee0a-0ff6-41d4-83a7-480f05229908.png)
how to change it ?

**Comment by yiheng-wang-nv** [comment, marked as answer]
Hi @jjhhan , you can use `scores = dice_metric_batch.aggregate()`, then `print(scores)` to see the results for each channel.
There is an example which also has multiple classes:
https://github.com/Project-MONAI/tutorials/blob/ff586e0cf24ac164553327b9453edab147cffa9e/3d_segmentation/brats_segmentation_3d.ipynb

**Reply by jjhhan** [reply]
Thanks a lot，I've solved it.   : )
